process/interface.go

```diff
@@ -168,28 +168,12 @@ type TransactionVerifier interface {
 
 // TransactionFeeHandler processes the transaction fee
 type TransactionFeeHandler interface {
+	CreateBlockStarted()
+	GetAccumulatedFees() *big.Int
 	ProcessTransactionFee(cost *big.Int)
 	IsInterfaceNil() bool
 }
 
-// SpecialAddressHandler responds with needed special addresses
-type SpecialAddressHandler interface {
```

Comment:
I think we will need this (or a subset) for the meta processing. Anyway, can be deleted now as we will recreate it in the final form when implementing meta processing.

Reply:
not needed, it is handled directly in peer processor.

Reply:
ok